sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/epkversion/PartitionProcessorImpl.java

```diff
@@ -248,7 +271,39 @@ public Mono<Void> run(CancellationToken cancellationToken) {
                                     }).flatMap(values -> Flux.empty());
                             }
                         }
-                        break;
+                        case PARSING_ERROR:
+                            if (this.unparseableDocumentRetries.incrementAndGet() == 1) {
+                                logger.warn(
+                                    "Lease with token {}: Attempting a retry on parsing error.",
+                                    this.lease.getLeaseToken());
+                                this.resultException = new RuntimeException(clientException);
+                                this.options.setMaxItemCount(1);
```

Comment:
I am curious, will this enforce a response of 1 item or is this just a hint to the backend?

Reply:
If this is a hint, how will we be able to skip this record and get a new continuation token if the backend keeps returning multiple items repeatedly?